### PR TITLE
[Translation zh-tw] Shared > Udacity > ud892.html 

### PR DESCRIPTION
**Pull request by limhenry**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/386)

### CODE REVIEW

**Comment by limhenry** [comment]
The translation is based on Udacity (Chinese) website
https://cn.udacity.com/course/web-tooling-automation--ud892

File: src/content/zh-tw/_shared/udacity/ud892.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
```
Comment (petele) [inline src/content/zh-tw/_shared/udacity/ud892.html]: original does not contain the doctype, `html` or `body` tag. These files are included in to other files and should not include them.

Comment (limhenry) [reply src/content/zh-tw/_shared/udacity/ud892.html]: cc @agektmr @sotayamashita

**Comment by petele** [comment]
@sotayamashita & @chikathreesix - would you please take a look!

**Pull request closed without merging**